packages/utils/src/dataEncryptionKey.ts

```diff
@@ -0,0 +1,53 @@
+import { ec as EC } from 'elliptic'
```

Comment:
Mostly just moving @trianglesphere's code from commentEncryption to this new file.
A few notable changes in the `derive` function below